### PR TITLE
Bumps CI config

### PR DESCRIPTION
**Pull request by ened**
This should resolve issues during PR review.

### CODE REVIEW

**Comment by ened** [comment]
This also reactivated the tests now. It seems the json mapper methods have been made private, but you can use `@visibleForTesting` instead.

File: lib/src/workmanager.dart [hunk 1]
```diff
@@ -256,8 +261,5 @@ class JsonMapperHelper {
   }
 
   static String _enumToString(final dynamic enumeration) =>
-      enumeration
-          ?.toString()
-          ?.split('.')
-          ?.last;
+      enumeration?.toString()?.split('.')?.last;
```
Comment (ened) [inline lib/src/workmanager.dart:264]: Explanation for this (and other) formatting changes: I usually format a file before saving - internally that is using dartfmt. Let me know if that is unacceptable here.

**Comment by ened** [comment]
@jeremie-movify FYI

File: lib/src/workmanager.dart (hunk 1 above)
Comment (jeremie-movify) [reply lib/src/workmanager.dart:264]: On the contrary, it's a welcome change 🙂

**Pull request merged**